app/services/agreements/pdf_fields.rb

```diff
@@ -62,27 +62,31 @@ def print_paragraph_number_3(type)
       @pdf.start_new_page if @pdf.cursor < 120
       @pdf.text '§ 3.', align: :center, style: :bold
       @pdf.move_down 5
-      @pdf.text I18n.t("apps.hardware.#{type}_agreement.paragraph_number_3")
+      I18n.t("apps.hardware.#{type}_agreement.paragraph_number_3", bullet: '•').split('<br>').each do |paragraph|
+        @pdf.text paragraph
+      end
       @pdf.move_down 30
     end
 
     def print_company(company)
-      @pdf.text company.name, style: :bold
-      @pdf.text company.address
+      @pdf.text company.name + ' Sp. z o.o.', style: :bold
```

Comment:
No need for that change as we will load company name from DB and adjust it there. Same with company.address